Catalog docs (#126)
intel/AiCSD: docs_src/catalog/Pipeline-Configuration.md

# Pipeline Configuration for Intel® Geti™ Platform Model
## Overview

Deploy Intel® Geti™ platform models on this reference implementation. 

> With the Intel® Geti™ platform, you can train your own models for object detection, classification, segmentation, and anomaly. Learn more about [Intel® Geti™ platform](https://geti.intel.com/).

Reviewer: Do we want this written in second person?

```suggestion
The [Intel® Geti™ platform](https://geti.intel.com/) can be used to train custom models for object detection, classification, segmentation, and anomaly detection.
```
